### DISCUSSION TITLE
Intermittent blurry text when reviewing code

### DISCUSSION THREAD

**Discussion by nfriedly** [community/community#68554] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Occasionally, when reviewing code on github.com, I'll notice that all or part of the webpage is blurry. You can see it in the screenshot below, it goes fuzzy in the middle of the word "github" on the top line, and on the bottom line "Checkout" is crisp, while "the repository" is blury.

<img alt='Cropped Screenshot' src='https://webcompat.com/uploads/2023/9/62b5052d-5b82-44df-a45a-d6425a831ce5.jpg'>

(You can download the original, without compression artifacts, from https://nc.home.nfriedly.com/s/f46SNqY39nyibEM)

I'm using a M1 Macbook Pro (macOS 13.6) and an LG 34BK95U-W monitor running at "Default" resolution scale (200% I think). I'm in Firefox 117 right now, but I'm pretty sure I've run across this with previous versions (I thought it was my eyes initially!)

I'm not sure if this is a Github issue or a Firefox issue, so I figured I'd report it to both (FF bug report at https://github.com/webcompat/web-bugs/issues/127662)

**Comment by Primus66** [comment]
- If you’re using GitHub Dark Mode, try switching back to the light mode temporarily to see if it affects text rendering.

- Disable browser extensions one by one to see if any of them are causing the blurriness. Some extensions might interfere with page rendering.

**Reply by nfriedly** [reply]
Thanks

* I'm in light mode; I have the computer switch automatically depending on the time of day. I don't recall it happening when in dark mode, but that might be just because this computer is mostly used during working hours when it's in light mode.

* I opened up the same URL in a private window with all extensions disabled and observed the same behavior.

**Reply by nfriedly** [reply]
(I tried forcing the system into dark mode but it's an intermittent issue and now I can't get it to occur again in dark or light mode.)

**Reply by Primus66** [reply]
Oh I think that has solved the problem.

You can as well mark my solution as an answer

**Reply by nfriedly** [reply]
LOL, I wish! It'll be back in a day or two.
